XpringKit/PayID/Generated/APIClient.swift

```diff
@@ -0,0 +1,252 @@
+//
+// Generated by SwagGen
+// https://github.com/yonaskolb/SwagGen
+//
+
+import Foundation
+import Alamofire
+
+/// Manages and sends APIRequests
+public class APIClient {
+
+    public static var `default` = APIClient(baseURL: "")
+
+    /// A list of RequestBehaviours that can be used to monitor and alter all requests
+    public var behaviours: [RequestBehaviour] = []
+
+    /// The base url prepended before every request path
+    public var baseURL: String
+
+    /// The Alamofire SessionManager used for each request
+    public var sessionManager: SessionManager
+
+    /// These headers will get added to every request
+    public var defaultHeaders: [String: String]
+
+    public var jsonDecoder = JSONDecoder()
+    public var jsonEncoder = JSONEncoder()
+
+    public var decodingQueue = DispatchQueue(label: "apiClient", qos: .utility, attributes: .concurrent)
+
+    public init(baseURL: String, sessionManager: SessionManager = .default, defaultHeaders: [String: String] = [:], behaviours: [RequestBehaviour] = []) {
+        self.baseURL = baseURL
+        self.sessionManager = sessionManager
+        self.behaviours = behaviours
+        self.defaultHeaders = defaultHeaders
+        jsonDecoder.dateDecodingStrategy = .custom(dateDecoder)
+        jsonEncoder.dateEncodingStrategy = .formatted(API.dateEncodingFormatter)
+    }
+
+    /// Makes a network request
+    ///
+    /// - Parameters:
+    ///   - request: The API request to make
+    ///   - behaviours: A list of behaviours that will be run for this request. Merged with APIClient.behaviours
```

Comment:
behaviours is a British/Commonwealth spelling. It's not wrong per se, but you might want to reconsider.

Reply:
It's probably not worth changing at this point.

Reply:
This is generated code, so we're at the mercy of the British engineer who forced this upon us.